Fix markdown conversion errors in native/guide/ui

journey2w opened the pull request:
### Bugs Fixed ###
- Fix markdown conversion errors in native/guide/ui
- Issue #265

tizen-build commented:
checkout the webpage https://docs.tizen.org/staging/294

mijin80lee commented on docs/application/native/guides/ui/efl/mobile/component-tooltip.md:
```diff
@@ -6,7 +6,7 @@ This feature is supported in mobile applications only.
 ## Basic Usage
 
-> **Note**  
+> **Note**
```
line 9: Note form was not applied.
line 75: Note form was not applied.

journey2w replied:
done.

mijin80lee commented on docs/application/native/guides/ui/efl/wearable/component-circle-spinner.md:
````diff
@@ -27,7 +27,9 @@ To create a circle spinner component:
+   Pass a [circle surface](component-circle-surface.md) as the second parameter.
 
    ```
    circle_spinner = eext_circle_object_spinner_add(spinner, surface);
````
line 48: The separated lines are merged as one line. (In the following example,)

journey2w replied:
done

mijin80lee commented on docs/application/native/guides/ui/efl/wearable/component-ctxpopup.md:
```diff
@@ -32,11 +32,14 @@ When shown, the ctxpopup automatically selects an area inside its parent object'
 - `select_mode`
-- `select_mode/top`, `select_mode/bottom`These 2 styles can be used as a pair.
+
+- `select_mode/top`, `select_mode/bottom`
```
line 41,42: The separated lines are merged as one line.(These 2 styles)
line 85: The separated lines are merged as one line. Code block.(elm_object_item_part_text_set(it, "default", "New label");)

journey2w replied:
I think there are some wrong code tags, but I didn't fix them because the author thought the descriptions were correct.

journey2w commented:
/native/guides/ui/efl/container-layout.md
fixed: line 170: The separated lines are merged as one line.

/native/guides/ui/efl/learn-edc-styles.md
fixed: line 33,35,37,39,41,43,45,47: Same level with the line 27.

/native/guides/ui/efl/mobile/component-genlist.md
fixed: line 87: The separated lines are merged as one line.
fixed: line 354: Note form was not applied to the line 367.

/native/guides/ui/efl/wearable/component-check.md
fixed: line 36: The separated lines are merged as one line.

/native/guides/ui/efl/wearable/component-datetime.md
fixed: line 37: This sentence should not be included in the Note.(In both these signals, the event_info callback parameter is NULL.)

/native/guides/ui/efl/wearable/component-entry.md
fixed: line 286,296: The separated lines are merged as one line.
fixed: line 349: This sentence should not be included in the Note.(If not mentioned separately, the event_info callback parameter in all signals is NULL.)

/native/guides/ui/efl/wearable/component-genlist.md
fixed: line 95: Add a space. (to the PARTparameter's theming)
fixed: line 114,115: The separated lines are merged as one line.
fixed: line 129,130: These two sentences should not be included in the Note.
(The elm_genlist_item_show() function scrolls the scroller to show the desired item as visible.)
(The elm_object_item_data_get() function returns the data pointer set by the item creation functions.)
fixed: line 161: The separated lines are merged as one line.

/native/guides/ui/efl/wearable/component-plug.md
fixed: line 47: Add 1 space at the beginning. ( ELM_WIN_SOCKET_IMAGE);)

/native/guides/ui/efl/wearable/component-radio.md
fixed: line 46: The separated lines are merged as one line.
fixed: This sentence should not be included in the Code block.
(The following example shows a simple use case of the radio component.)

/native/guides/ui/efl/wearable/component-win.md
fixed: line 13: Broken image. (Win)
line 32: The separated lines are merged as one line.
=> There is no problem.

line 48: Add one blank line.
=> Don't need a blank line here.

fixed: line 62: Add '}".
fixed: line 97: Add 1 space. (with the evas_object_size_hint_weight_set()function)
line 132: Remove 's'. ( by the Windows Manager.)

/native/guides/ui/efl/wearable/ui-components.md
fixed: line 43: Check the 'elm-datetime'. (elm_datetime?!)
fixed: line 45: The separated lines are merged as one line.(Circular components can usually be)
fixed: line 67,68: Add '-' in the blank.
fixed: line 110,111,112,113: The separated lines are merged as one line.
fixed: line 199: The separated lines are merged as one line.
fixed: line 202,203,206,208: The separated lines are merged as one line.

journey2w commented on docs/application/native/guides/ui/dali/index.md:
```diff
@@ -52,11 +52,15 @@ DALi has a concept of a camera to display its virtual 3D world to a 2D screen. T
+
 - **DALi Toolkit**
-This module provides UI components and various effects on top of the DALi core.
+  This module provides UI components and various effects on top of the DALi core.
```
done

journey2w commented on docs/application/native/guides/ui/efl/accessibility-implementation.md:
```diff
@@ -262,9 +262,17 @@ The Elementary library sets the default information (label, traits, state) for t
-- `on&off` style in a checkbox:The reading must be "On/off button, <state>", where <state> is read as "on" or "off", depending on the checkbox state.
+- `default` style in a checkbox:
+
+   The reading must be "<label>, <state>", where <label> is a text assigned to the checkbox and <state> is read as "selected" or "not selected", depending on the checkbox state.
```
done.
\<

\< is `\<` and  in \> is `\>`.

journey2w commented on docs/application/native/guides/ui/efl/component-custom.md:
````diff
@@ -293,8 +298,13 @@ To create a customized button component style:
+   >
+   > In the EDC file, programs are executed by the `after` keyword or the `run_program` script function. For example:
+   > ```
+   > after: "touch_sound"; /* OR */
````
done

journey2w commented on docs/application/native/guides/ui/efl/data-types.md:
```diff
@@ -110,7 +112,9 @@ To manage string buffers:
        eina_strbuf_append_char(mybuffer, 'r');To handle "printf" format strings, use the `eina_strbuf_append_printf()` function to add formatted strings to the buffer:`eina_strbuf_append_printf(buf, "%s%c", "buffe", 'r');`
```
done

journey2w commented on docs/application/native/guides/ui/efl/ecore-animation.md:
```diff
@@ -59,26 +60,13 @@ _my_animation(void *data, double pos)
 >
 > The callback function receives a timeline position (second parameter) with a value between 0.0 (start) to 1.0 (end) to indicate where along the timeline the animator is running.
 
 ## Starting an Infinite Animation
```
done

journey2w commented on docs/application/native/guides/ui/efl/edje-animation.md:
```diff
@@ -123,9 +124,18 @@ The following types of transitions can be added to the programs to create effect
+  - Number of spring swings
 
 For graphical representations of these effects, see the [position mappings in Ecore animations](ecore-animation.md#table_position_mappings).
 
```
done

journey2w commented on docs/application/native/guides/ui/efl/efl-util.md:
```diff
@@ -123,8 +123,8 @@ To create notification windows and access the current notification level of an e
 
   - To create a new window and give it the `NOTIFICATION` type, call the `elm_win_add()` function with the third parameter set to `ELM_WIN_NOTIFICATION`:  
```
done

journey2w commented on docs/application/native/guides/ui/efl/elementary-animation.md:
```diff
@@ -315,7 +321,8 @@ To add effects for different transition types:
+  >
   > Do not use this function to hide an Evas object by setting one of the length attributes to zero. Use the wipe effect described above instead.
 
   The following example shows how to make a rectangle exchange its width and height properties in a 2-second transition:
```
There is no problem on line 345.

journey2w commented on docs/application/native/guides/ui/efl/event-types.md:
````diff
@@ -167,41 +167,43 @@ The following Ecore events are available for shortcuts. The event callbacks rece
+   ```
 
 ### Sending Ecore Events to the Main Loop
 
````
done.

journey2w commented on docs/application/native/guides/ui/efl/font-setting.md:
```diff
@@ -49,15 +49,15 @@ For example:
-      font_size: 36;      
-      text_class: "tizen";   
+  description {
+    text {
```
done

journey2w commented on docs/application/native/guides/ui/efl/learn-edc-program.md:
```diff
@@ -81,13 +81,16 @@ program {
-  - `PLAY_SAMPLE "sample name" speed (channel)`: Play a music sample clip`PLAY_SAMPLE`'s (optional) channel can be one of:
+  - `PLAY_SAMPLE "sample name" speed (channel)`:
+
+    Play a music sample clip`PLAY_SAMPLE`'s (optional) channel can be one of:
```
done.

journey2w commented on docs/application/native/guides/ui/efl/scalability.md:
```diff
@@ -173,7 +174,9 @@ Parts are used to represent the most basic layout elements, such as a line in a
 The part blocks in the EDC file can have the following property related to scalability:
 
-- `scale`: 0-1Specifies whether the part scales its size according to the scaling factor (Tizen has a scaling factor to resize the application layout and object). This property is used to scale object properties, such as font size or min/max size of the part.The default value is 0 (off) and the default scaling factor is 1.0. To make a part scalable, set the property to 1 (on).
+- `scale`: 0-1Specifies whether the part scales its size according to the scaling factor (Tizen has a scaling factor to resize the application layout and object). This property is used to scale object properties, such as font size or min/max size of the part.
```
It is not line 17 but line 117.
line 298: fixed.

journey2w commented on docs/application/native/guides/ui/efl/mobile/component-datetime.md:
```diff
@@ -136,12 +138,13 @@ The following table lists the available component styles.
 | Style                  | Sample                                   |
-|------------------------|----------------------------------------|
+|------------------------|------------------------------------------|
 | `default``date_layout` | ![elm/datetime/base/default](./media/datetime_date_layout.png) |
```
done.

journey2w commented on docs/application/native/guides/ui/efl/mobile/component-index.md:
````diff
@@ -148,13 +148,16 @@ The following table lists the available component styles.
+> elm_index_horizontal_set(index, EINA_TRUE);
+> ```
 
 For the `default` style, both orientations are supported, but for the `pagecontrol` style, only horizontal mode is available.
````
done

journey2w commented on docs/application/native/guides/ui/efl/mobile/component-map.md:
```diff
@@ -107,7 +107,7 @@ You can register callback functions connected to the following signals for a map
 **Table: Map callback signals**
```
done.

journey2w commented on docs/application/native/guides/ui/efl/mobile/component-slider.md:
```diff
@@ -175,7 +175,7 @@ The following table lists the available component styles.
 | Style          | Horizontal                               | Vertical                                 |
-|-------------|----------------------------------------|----------------------------------------|
+|----------------|------------------------------------------|------------------------------------------|
 | `default`      | ![elm/slider/horizontal/default](./media/slider_hor.png) | ![elm/slider/vertical/default](./media/slider_ver.png) |
```
done.

mijin80lee replied:
line 179: Broken images. 
![elm/slider/horizontal/default](./media/slider_hor.png)
![elm/slider/vertical/default](./media/slider_ver.png)

journey2w replied:
done

journey2w commented on docs/application/native/guides/ui/efl/mobile/component-progressbar.md:
````diff
@@ -127,12 +127,14 @@ To configure the progressbar features:
-  /* Supported styles: default, pending */elm_object_text_set(progressbar, "progressbar");
-  ```
+    ```
+    /* Supported styles: default, pending */elm_object_text_set(progressbar, "progressbar");
````
done.

tizen-build commented:
checkout the webpage https://docs.tizen.org/staging/294

tizen-build commented:
checkout the webpage https://docs.tizen.org/staging/294

mijin80lee commented on docs/application/native/guides/ui/efl/mobile/component-entry.md:
````diff
@@ -185,7 +192,10 @@ To manage the cursor position:
-- Set the cursor at a specific position (15th character, for example):`elm_entry_cursor_pos_set(entry, 15);`
+- Set the cursor at a specific position (15th character, for example):
+  ```
+  elm_entry_cursor_pos_set(entry, 15);`
````
line 197: Remove '`' at the end.

journey2w replied:
done

journey2w commented on docs/application/native/guides/ui/efl/accessibility-implementation.md:
```diff
@@ -262,9 +263,17 @@ The Elementary library sets the default information (label, traits, state) for t
-- `on&off` style in a checkbox:The reading must be "On/off button, <state>", where <state> is read as "on" or "off", depending on the checkbox state.
+- `default` style in a checkbox:
+
+   The reading must be "\<label\>, \<state\>", where \<label\> is a text assigned to the checkbox and <state> is read as "selected" or "not selected", depending on the checkbox state.
```
When you write a comment about wrong usage, you can use code tag to show the current state.
For instance,
I can't see the `<state>`, if I write like below:
line 268: Rewrite '<state>' to '\<state\>'.

I can see what is wrong exactly, if you write like below:
line 272: Rewrite `'<state>'` to '\<state\>'.

Thanks.

journey2w commented on docs/application/native/guides/ui/efl/ecore-animation.md:
```diff
@@ -194,18 +198,119 @@ The following table lists the supported position mappings, and describes how the
+		<tr>
+			<td><code>ECORE_POS_MAP_BOUNCE</code></td>
+			<td>Start at 0.0, then drop like a ball bouncing to the ground at 1.0, and bounce <code>v2</code> times, with a decay factor of <code>v1</code>
+			<p><img alt="Position mappings" src="/sites/default/files/images/pos_map_bounce.png" /></p>
```
done

tizen-build commented:
checkout the webpage https://docs.tizen.org/staging/294

tizen-build commented:
checkout the webpage https://docs.tizen.org/staging/294
